packages/react-core/src/components/Tooltip/Tooltip.tsx

```diff
@@ -60,7 +68,22 @@ export interface TooltipProps extends Omit<React.HTMLProps<HTMLDivElement>, 'con
    * ['top', 'right', 'left']. Since there is no space to the top, it checks if right is available. There's also no
    * space to the right, so it finally shows the tooltip on the left.
    */
-  flipBehavior?: 'flip' | ('top' | 'bottom' | 'left' | 'right')[];
+  flipBehavior?:
+    | 'flip'
+    | (
```

Comment:
I think this would read better if abstracted into its own type (everything in the parentheses)

Reply:
could, but i think we'd still need to type it out for the parser anyway... @nicolethoen ?

Reply:
You could test it to be sure. But that certainly has been the case in the past.